### PR TITLE
Added Documentation to utils.py

### PR DESCRIPTION
**Pull request by mmessick**
@Amy-Xu, hoping you can take a look at some of the documentation I provided for the last few functions in the utils.py file. I tried to match the same formatting you used. Let me know if I didn't understand the functions properly or need to go into more detail about them

### CODE REVIEW

**Comment by Amy-Xu** [comment]
Good job! It looks good to me. I added my suggestion in two line notes. @MattHJensen Could you review this PR?

File: taxcalc/utils.py [hunk 1]
```diff
@@ -272,24 +274,31 @@ def add_weighted_decile_bins(df):
...
-    df: DataFrame to group
 
-    compare_with: string, optional
-            Some names to specify certain pre-defined bins
+    Parameters
+    ----------
+    df: DataFrame object
+
+    compare_with: String, optional
+        Some names to specify certain pre-defined bins
```
Comment (MattHJensen) [inline taxcalc/utils.py]: Do you think it would be helpful to list the `compare_with` options here?

**Comment by MattHJensen** [comment]
@mmessick this looks great! I had a minor question, above. Take a look at that, look over everything once more, and when you are ready let me know and I'll merge this in.

**Comment by mmessick** [comment]
@MattHJensen looks like it should be good to merge now

**Comment by MattHJensen** [comment]
Merging. Thanks, @mmessick.

**Pull request merged**